mx.py

```diff
@@ -1791,7 +1791,7 @@ def getBuildTask(self, args):
         return JARArchiveTask(args, self)
 
     def exists(self):
-        return exists(self.path) and not self.sourcesPath or exists(self.sourcesPath)
+        return exists(self.path) and not self.sourcesPath or exists(self.sourcesPath) #pylint: disable=consider-using-ternary
```

Comment:
I felt this statement made more sense in it's current form then when using the ternary (`not self.sourcesPath if exists(self.path) else exists(self.sourcesPath)`) since we actually want to do a logical and. So I've disabled the warning.